inst/extdata/emissions/GFED/GFED_ForestFire_CO.csv

```diff
@@ -0,0 +1,250 @@
+# File: GFED_ForestFire_CO.csv,,,,,,,,,,,,,,,,,,,

```

Comment:
Please remove ugly Excel trailing commas from header lines, here and in all files